### ISSUE TITLE
Breaks chat and follow option in nodebb when enabled

### ISSUE THREAD

**Issue by Nisthar** [reddink/nodebb-plugin-reddcoin#4]
Breaks chat and follow option in nodebb when the reddcoin plugin is enabled. Also its changing my theme from persona to lavender in the user profile section when this plugin is enabled. Any solutions?

Was a nice plugin. I had to disable it to make my chat and follow option working.

**Comment by reddink** [comment]
What version of nodebb are you running?
When did it stop working for you?

**Comment by Nisthar** [comment]
I am running 0.9.3 version.It breaks the chat and follow option if i enable the plugin and change my profile section to lavender theme, i think

**Comment by reddink** [comment]
Thanks for the feed back.
I haven't tested yet on 0.9.3. Let me get it into my dev cycle

**Comment by Nisthar** [comment]
@reddink Its a nice plugin. Waiting for the fix.

**Comment by reddink** [comment]
Just noticed nodebb has released version 1.0
Need to catch up with versions.
Do you have a plan to update in the short term?

**Comment by Nisthar** [comment]
Yes, I'll update it soon. But i think there are no major difference in the api of 1.0.0 and 0.9.4.

**Comment by reddink** [comment]
Sorry for the long delay on this,
Dev cycle has come around :)
I am in the process of testing scenarios.
Just as a heads up, I upgraded my local nodebb to v1.1.0 with the lavender theme

One issue I found was the version compatibility plugin was set to v0.9.0
I have made an update for compatibility v1.0.0 and updated the release of the plugin to v0.1.5

If you're able, could you update the plugin and provide feedback.
Seems to be working ok (on my system)
